GUI/AllegroInput.cpp

```diff
@@ -307,28 +357,6 @@ void AllegroInput::Update(void)
         mouse_x = MIN((g_FrameMan.GetResX() * mouseDenominator) - 3, mouse_x);
         mouse_y = MIN((g_FrameMan.GetResY() * mouseDenominator) - 3, mouse_y);
 
-        // Button states/presses, Primary - ACTUALLY make either button work, we don't have use for secondary in menus
```

Comment:
all this block is deleted because it doesn't seem to have any positive effect, on the contrary it has a negative effect of changing mouse states that were set earlier in this same function.

Reply:
wrong. fixed it.